src/components/WwwFrame/PromotionalBanner/Banners/PromoCreditBanner.vue

```diff
@@ -158,25 +179,17 @@ export default {
 	},
 	methods: {
 		fetchManagedAccountCampaign() {
-			// TODO: Create an array of queries if multiple credits have an associated promo fund
-			// extended promotion information from managed account
-			if (this.promoFundId && this.hasFreeCredits && this.creditAvailable) {
-				this.apollo.query({
-					query: promoCampaignInfo,
-					variables: {
-						promoFundId: String(this.promoFundId)
-					}
-				})
-					.then(({ data }) => {
-						this.promoCampaignData = data?.shop?.promoCampaign ?? null;
-					});
-			}
+			this.apollo.query({
+				query: promoCampaignInfo,
+				variables: this.promoFundId ? {
+					promoFundId: String(this.promoFundId)
+				} : {}
+			})
+				.then(({ data }) => {
+					this.promoCampaignData = data?.shop?.promoCampaign ?? null;
+				});
 		},
 		setPromoState(promotionData) {
-			// Parse user promoBalance and creditAvailableTotal from basket
-			const userPromoBalance = promotionData?.my?.userAccount?.promoBalance ?? 0;
-			const basketPromoBalance = promotionData?.shop?.basket?.totals?.creditAvailableTotal ?? 0;
```

Comment:
Giving precedent to this was allowing this banner to show for Kiva Credit too. While it's an interesting to thought to consider a more visual reminder we'll deal with it elsewhere and at another time.